Angular/src/app/app.module.ts

```diff
@@ -90,6 +90,7 @@ import { OverviewComponent } from './overview/overview.component';
 import { IntroductionsComponent } from './dashboard/introductions/introductions.component';
 import { ExpandableTextComponent } from './other_components/expandable-text/expandable-text.component';
 import { DocumentFilterComponent } from './dialogs/document-filter/document-filter.component';
+import { UsersComponent } from './dashboard/users/users.component';
```

Comment:
Is t niet logischer om deze in dashboard.component.ts te importen? Zelfde kan overigens ook over mijn introductions gezegd worden, dus mss moet die ook maar even naar dashboard halen. Had m in 1e instantie hier geplaatst omdat ik mss iets met de introduction dialog op application-level wou doen, maar is denk ik niet meer nodig,

Reply:
Elk component moet gedeclareerd worden in app.module.ts :) Zo weet Angular dat een component bestaat!

Reply:
Ah, natuurlijk, met de declarations, I knew that. T viel me voornamelijk op dat in dashboard.component de andere components daar wel geimport worden en de userscomponent nog niet

Reply:
Oh ja. Ik snapte al niet waarom de Introductions en bibliography op die manier geimporteerd waren in t Dashboard, want dat is helemaal niet nodig aangezien users en introductions volledig onafhankelijke components zijn. Ik had het al weggehaald uit mn nieuwe Usercomponent, maar nog niet uit Dashboard blijkbaar! Nu gefixed :D

Reply:
Zie t! Zal m mergen!